ams/server/src/main/java/com/netease/arctic/server/optimizing/KeyedTableCommit.java

```diff
@@ -75,8 +75,7 @@ public void commit() throws OptimizingCommitException {
     Set<DeleteFile> addedDeleteFiles = Sets.newHashSet();
     Set<DeleteFile> removedDeleteFiles = Sets.newHashSet();
 
-    StructLikeMap<Long> partitionOptimizedSequence =
-        TablePropertyUtil.getPartitionOptimizedSequence(table.asKeyedTable());
+    StructLikeMap<Long> partitionOptimizedSequence = PuffinUtil.reader(table.asKeyedTable()).readOptimizedSequence();
```

Comment:
Should we can wrap PuffinUtil.reader(table.asKeyedTable()).readOptimizedSequence() logic into KeyedTable interface?

Reply:
I think it is not suitable to include this in the `KeyedTable` interface. We do not want to expose the `optimized sequence` to the users of the `Table` API, even though the current encapsulation is not yet perfect.

BTW, I have move the `PuffinUtil.reader(table.asKeyedTable()).readOptimizedSequence()` to `com.netease.arctic.utils.ArcticTableUtil#readOptimizedSequence`.